tests/ut_callmodel/callmodeltest.cpp

```diff
@@ -379,7 +372,7 @@ void CallModelTest::testDeleteEvent()
     model.setTreeMode(true);
     QVERIFY( model.setFilter( CallModel::SortByContact ) );
     QVERIFY( model.getEvents() );
-    QVERIFY(watcher.waitForModelReady());
+    QTRY_VERIFY(modelReady.count() > 0); modelReady.clear();
```

Comment:
why > 0 rather than == 1 ?  does setFilter() and getEvents() both cause the signal to be emitted?  (I assume its those two, since above there's a setTreeMode() and getEvents() with QTRY_COMPARE(modelReady.count(), 1)).  If so, can we compare == 2 and add a comment to say why we expect 2?

Reply:
This was more just replacing the original watcher.waitForModelReady() which didn't specify the expected signal count. 

setFilter() may cause a getEvents() to be done; if so, then modelReady() is emitted twice, otherwise we would only expect it to be emitted once here. I've added a commit to change these to wait for either 1 or 2 signals.